Source/WebCore/css/parser/CSSParserFastPaths.cpp

```diff
@@ -150,17 +150,35 @@ static inline bool parseSimpleAngle(std::span<const CharacterType> characters, C
     return parsedNumber.has_value();
 }
 
-static RefPtr<CSSValue> parseSimpleLengthValue(CSSPropertyID propertyId, StringView string, CSSParserMode cssParserMode)
+template <typename CharacterType>
+static inline bool parseSimpleNumberOrPercentage(std::span<const CharacterType> characters, ValueRange valueRange, CSSUnitType& unit, double& number)
```

Comment:
We don't have great documentation of these functions, but a comment indicating what differentiates "simple" from normal here would be quite useful for readers. 

(like above, changing to a std::optional return would be a bit more idiotic, but also like above, not worth doing with this functional change).

Reply:
Agree. Autogenerating more of this would be great.